src/librustc_mir/util/pretty.rs

```diff
@@ -607,7 +607,7 @@ pub fn write_allocations<'tcx>(
                     Ok(_) => {
                         span_bug!(tcx.def_span(did), " static item without `ByRef` initializer")
                     }
-                    Err(_) => writeln!(w, " error during const eval of static initializer")?,
+                    Err(_) => write!(w, " error during const eval of static initializer")?,
```

Comment:
So this would look line
```
alloc13 error during const eval of static initializer
```
which seems not great. Maybe
```
alloc13 (static: X, evaluation error)
```
or so?